client/src/helpers/APICalls/login.ts

```diff
@@ -1,6 +1,22 @@
-import { AuthApiData } from '../../interface/AuthApiData';
+import { AuthApiData, DemoAuthData } from '../../interface/AuthApiData';
 import { FetchOptions } from '../../interface/FetchOptions';
 
+export const demoLogin = async (email: string, password: string): Promise<DemoAuthData> => {
+  const fetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ email, password }),
+  };
```

Comment:
We should be setting the base of our branch correctly so we don't see duplicated code.

Reply:
Please what do you mean by updating the base of the branch?

Reply:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request